src/validations/rules/Writing_FedRAMP_Validations.md

```diff
@@ -0,0 +1,375 @@
+# How to create FedRAMP OSCAL validation constraints
+
+FedRAMP OSCAL documents are structured XML documents. The document syntax is defined by OSCAL XML Schema. However, XML Schema cannot express many semantic constraints, and an additional schema language is employed: Schematron.
```

Comment:
It may be overkill but I would define Schematron via "Schematron is used for business rules validation, data reporting, general validation, quality control, quality assurance, firewalling, filtering, contraint checking, naming and design rules checking...."   - from the Schematron website.

Reply:
An abbreviated description of (FedRAMP-specific) use will be added.